src/script/conversation/ConversationRepository.js

```diff
@@ -2755,17 +2755,21 @@ export class ConversationRepository {
       const allRecipientsBesideSelf = Object.keys(eventInfoEntity.options.recipients).filter(
         id => id !== this.selfUser().id,
       );
+      const userIdsWithoutClients = [];
       for (const recipientId of allRecipientsBesideSelf) {
         const clientIdsOfUser = eventInfoEntity.options.recipients[recipientId];
         const noRemainingClients = clientIdsOfUser.length === 0;
 
         if (noRemainingClients) {
-          const backendUser = await this.userRepository.getUserFromBackend(recipientId);
-          const isDeleted = backendUser?.deleted === true;
+          userIdsWithoutClients.push(recipientId);
+        }
+      }
+      const userList = await this.userRepository.getUsersById(userIdsWithoutClients);
+      for (const user of userList) {
+        const isDeleted = user?.deleted === true;
```

Comment:
Shouldn't it be `user?.isDeleted`?

Reply:
I think that's a value from the backend. Either way it's just what it was before :>

Reply:
Mh, looking a bit deeper in the code...this stuff seems to be pretty broken ;P